tensorboard/webapp/widgets/histogram/histogram_util_test.ts

```diff
@@ -276,8 +276,8 @@ describe('histogram util', () => {
           )
         ).toEqual([
           [
-            {x: 0, dx: 5, y: 100},
-            {x: 5, dx: 5, y: 100},
+            {x: 0, dx: 5, y: 0},
+            {x: 5, dx: 5, y: 200},
```

Comment:
Removed this test case since it's basically duplicate with the newly edited test suggested above (`produces result bins from multiple 0 width bins in different steps`).

Reply:
Oops, I actually commented in the wrong place - I meant to comment on the test name `redistributes 0 width bin evenly over edges of result bins` (line 263) but GH won't let me comment there.  It's the "evenly" that isn't accurate any more.

Reply:
Got it! Renamed.